students/kulon_aleksander/lesson_10/filling_in_the_gaps.py

```diff
@@ -0,0 +1,29 @@
+import re
+import os
+import shutil
+
+pattern = 'spam00'
```

Comment:
Because of that program will be not working correctly for files such as spam010.txt or spam100.txt. Shouldn't you consider such files as well? Maybe try to use better regex 😄